File: Solutions/1619. Mean of Array After Removing Some Elements/1619. Mean of Array After Removing Some Elements.java
class Solution {
    public double trimMean(int[] arr) {
        Arrays.sort(arr);
        double sum=0;
        int count=0,size=arr.length;
        for (int i = size / 20; i < size - (size / 20); ++i) {
		sum += arr[i];
		++count;
	}
        return sum / count;
    }
}

File: Solutions/1619. Mean of Array After Removing Some Elements/1619. Mean of Array After Removing Some Elements.go
func trimMean(arr []int) float64 {
    sort.Ints(arr)
    sum,count:=0,0
    size:=len(arr)
    for i:=size/20;i<size-size/20;i+=1{
        count+=1
        sum+=arr[i]
    }
    return float64(sum)/float64(count)
}